.github/workflows/test.yml

```diff
@@ -30,7 +28,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        composer require "illuminate/container:${{ matrix.illuminate_version }}" "illuminate/contracts:${{ matrix.illuminate_version }}" "illuminate/database:${{ matrix.illuminate_version }}" "illuminate/http:${{ matrix.illuminate_version }}" "illuminate/support:${{ matrix.illuminate_version }}" --no-interaction --no-progress --prefer-dist ${{ matrix.composer_flags }}
+        run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction
```

Comment:
Won't this ignore the `illuminate_version` above and just install the lowest `6.x` for `--prefer-lowest` and the highest `9.x` for `--prefer-stable`?